config.go

```diff
@@ -166,7 +166,7 @@ func (c *srcfileConfig) apply() error {
 	}
 
 	if config.GOROOTForCmd == "" {
-		config.GOROOTForCmd = config.GOROOT
+		config.GOROOTForCmd = buildContext.GOROOT
```

Comment:
Actually, `GOROOTForCmd` was introduced in 8ad0bf4df4faaa689afb7447358064ff4e7e8bcb, it looks like to handle the fact that Go 1.3 had standard packages in $GOROOT/src/pkg and 1.4+ has them in $GOROOT/src.

That will not be needed for 1.4, 1.5 and later, so perhaps we should remove `GOROOTForCmd` altogether since it's no longer needed. It will make things simpler.